lxd/main.go

```diff
@@ -780,7 +799,7 @@ func cmdInit() error {
 			return fmt.Errorf("Init configuration is only valid with --auto")
 		}
 
-		storageBackend = askChoice("Name of the storage backend to use (dir or zfs): ", backendsSupported)
+		storageBackend = askChoice("Name of the storage backend to use (dir or zfs) [default=zfs]: ", backendsSupported, "zfs")
```

Comment:
We'll probably want to do something a bit more clever here on systems without zfs support, but that can wait for when we do a bigger re-work of the init command around the time we land the new network API work.